client/state/posts/utils.js

```diff
@@ -31,7 +31,9 @@ import stripHtml from 'lib/post-normalizer/rule-strip-html';
  * Constants
  */
 
-const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/,
```

Comment:
I don't know, I've not used this RegExp, I haven't touched it but I'll check and see what's its purpose.